### ISSUE TITLE
Implement Audit Log

### ISSUE THREAD

**Issue by arw180** [ozone-development/ozp-backend#61]
The production system will require an audit log. Things that need to be audited:

* any changes to the database (note the change, user, and date/time)
* ?

Possible add-ons to use:
* http://django-reversion.readthedocs.org/en/latest/index.html
* https://github.com/Atomidata/django-audit-log

**Comment by tonyalletag** [comment]
ozp.log should have a time stamp and data should be in JSON format.

This is all we need for now.  Once this is done we can close the ticket. Another ticket can be open if we get additional logging request

**Comment by mannyrivera2010** [comment]
We were confused with this ticket, Can you provide more details? @arw180
